src/interface/src/app/services/map.service.ts

```diff
@@ -104,6 +105,8 @@ export class MapService {
         return 'north_coast_inland';
       case Region.SOUTHERN_CALIFORNIA:
         return 'southern_california';
+      case null:
+        return '';
```

Comment:
Should this default to Sierra Nevada (see two lines below)?

Reply:
Ooh thank you for catching this — fixed